lstchain/scripts/onsite/onsite_create_calibration_file.py

```diff
@@ -154,7 +154,7 @@ def main():
     # search the pedestal file of the same date
     if ped_run is None:
         # else search the pedestal file of the same date
-        file_list = sorted(Path(f"{ped_dir}/{date}/{pro}/").rglob(f'drs4_pedestal*.0000.h5'))
+        file_list = sorted(Path(f"{ped_dir}/{date}/{pro}/").rglob('drs4_pedestal*.0000.h5'))
```

Comment:
what is the difference computation-wise between using :
`f'drs4_pedestal*.0000.h5'` and `'drs4_pedestal*.0000.h5'`

Reply:
An f string without something to format is just a string